From: Colin Watson <cjwatson@ubuntu.com>
Date: Mon, 13 Jan 2014 12:13:27 +0000
Subject: Adjust efi_distributor for some distributions

This is not a very good approach, and certainly not sanely upstreamable;
we probably need to split GRUB_DISTRIBUTOR into a couple of different
variables.

Bug-Ubuntu: https://bugs.launchpad.net/bugs/1242417
Bug-Debian: https://bugs.debian.org/932966
Forwarded: not-needed
Last-Update: 2019-08-06

Patch-Name: install-efi-adjust-distributor.patch
---
 util/grub-install.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/util/grub-install.c b/util/grub-install.c
index 5ddd028..3a06718 100644
--- a/util/grub-install.c
+++ b/util/grub-install.c
@@ -1123,6 +1123,10 @@ main (int argc, char *argv[])
       */
       char *t;
       efi_distributor = bootloader_id;
+      if (strcmp (efi_distributor, "kubuntu") == 0)
+	efi_distributor = "ubuntu";
+      else if (strcmp (efi_distributor, "devuan") == 0)
+	efi_distributor = "debian";
       switch (platform)
 	{
 	case GRUB_INSTALL_PLATFORM_I386_EFI:
